gulp/Documentation.js

```diff
@@ -12,7 +12,12 @@ var folderList = Utilities.getFolders(Config.paths.componentsPath);
 var demoPagesList = Utilities.getFolders(Config.paths.srcDocsJSCompPages);
 var reload = require('require-reload')(require);
 var BuildConfig = require('./modules/BuildConfig');
+var production = process.argv.indexOf('--production') >= 0;
```

Comment:
This is super minor, but [`office-ui-fabric-react` uses `isProduction` for this flag](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/gulpfile.js#L10).